web/views/datasets.py

```diff
@@ -56,7 +126,8 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['is_admin'] = self.request.user.is_admin_of_dataset(self.object)
         context['can_edit'] = self.request.user.can_edit_dataset(self.object)
-        context['can_see_protected'] = self.request.user.has_permission_on_object(f'core.{Permissions.PROTECTED.value}_dataset', self.object)
+        context['can_see_protected'] = self.request.user.has_permission_on_object(
+            f'core.{Permissions.PROTECTED.value}_dataset', self.object)
```

Comment:
Usually in our projects, if the parameters of a function call are on a new line, the closing parenthesis should be on another new line, aligned with the function call.

E.g:
```python
method(
    param1, param2, param3
)
```

Reply:
Actually, ignore this.
We will run black after the sprint anyway.